cat/c11.cat

```diff
@@ -15,18 +15,22 @@ C "C++11"
 include "c11_cos.cat"
 include "c11_los.cat"
 
+// dynamic_tag relates events to itself that access an address whose init event is marked static_tag
+let dynamic_NAL = [range([IW & NAL]; loc)]
+let dynamic_AL = [range([IW & ~NAL]; loc)]
+
 let asw = IW * (M \ IW)
 let sb = po
-let mo = co & (~NAL * ~NAL)
+let mo = dynamic_NAL; co; dynamic_NAL
```

Comment:
shouldn't this be `dynamic_AL`?

I think it would be better to define a parametric definition
```
// dynamic_tag relates events to itself that access an address whose init event is marked X
let dynamic_tag(X) = [range([IW & X]; loc)]
```

Reply:
Updated